rt_gene/scripts/estimate_gaze.py

```diff
@@ -44,6 +43,17 @@ def __init__(self, device_id_gaze, model_files):
 
         self.tf_prefix = rospy.get_param("~tf_prefix", "gaze")
         self.headpose_frame = self.tf_prefix + "/head_pose_estimated"
+        self.ros_tf_frame = rospy.get_param("~ros_tf_frame", "kinect2_ros_frame")
```

Comment:
This should disappear ..

Reply:
sorry yes, amateur rebasing hour

Reply:
sorry again, rebasing issue. Thank you for catching.